exporter/awsemfexporter/util.go

```diff
@@ -131,13 +131,13 @@ func dedupDimensions(dimensions [][]string) (deduped [][]string) {
 // The returned dimensions are sorted in alphabetical order within each dimension set
 func dimensionRollup(dimensionRollupOption string, labels map[string]string) [][]string {
 	var rollupDimensionArray [][]string
-	var dimensionZero []string
+	dimensionZero := make([]string, 0)
```

Comment:
Is this necessary?  This forces an allocation even if nothing is ever put into this slice.  It looks like this may later be appended to the `rollupDimensionArray` slice.  Is the consumer of that slice unable to deal with `nil` values?

Reply:
this is more like a constant. empty dimension must be present in the dimension set array after the dimension roll up operation. 
I added comments and use literal initialization to make it more explicit.